bin.src/certifyCalibration.py

```diff
@@ -66,12 +75,18 @@
 
     butler = Butler(args.root, run=args.inputCollection)
 
+    # I'm not sure this is the best way to convert strings to Time objects, but
+    # it works fine on the bare dates we seem to be passing in these days, and
+    # I imagine it should work with more complete time strings as well.
+    timespan = Timespan(
+        begin=astropy.time.Time(args.beginDate) if args.beginDate is not None else None,
+        end=astropy.time.Time(args.endDate) if args.endDate is not None else None,
+    )
+
     # Do the thing.
-    certify = CertifyCalibration(butler=butler,
+    certify = CertifyCalibration(registry=butler.registry,
                                  inputCollection=args.inputCollection,
-                                 outputCollection=args.outputCollection)
+                                 outputCollection=args.outputCollection,
+                                 lastRunOnly=not args.lastRunOnly)
```

Comment:
This is a bit confusing, but I see it's because of the user-facing option and the CertifyCalibration option having different meanings.

Reply:
You were right to be confused; it's wrong, and it's how I broke this script (I added the command-line option after I tested ci_cpp_gen3 with the value defaulted to `True` in the task).  Fixing it now.